git-config/src/file/git_config.rs

```diff
@@ -131,8 +157,10 @@ impl<'event> GitConfig<'event> {
     /// Returns an error if there was an IO error or if the file wasn't a valid
     /// git-config file.
     #[inline]
-    pub fn open<P: AsRef<Path>>(path: P) -> Result<Self, ParserOrIoError<'static>> {
-        parse_from_path(path).map(Self::from)
+    pub fn open<P: AsRef<Path>>(path: P) -> Result<Self, from_paths::Error> {
```

Comment:
I don't thinks `open(…)` should use `from_paths::Error` - it doesn't do any include resolution and to my mind seems unrelated.
Maybe I am missing something.

Reply:
that's right, I went too far with it, rolled back!